store/sqlstore/session_store.go

```diff
@@ -102,18 +98,16 @@ func (me SqlSessionStore) Get(sessionIdOrToken string) (*model.Session, *model.A
 func (me SqlSessionStore) GetSessions(userId string) ([]*model.Session, *model.AppError) {
 	var sessions []*model.Session
 
-	tcs := me.Team().GetTeamsForUser(userId)
 	if _, err := me.GetReplica().Select(&sessions, "SELECT * FROM Sessions WHERE UserId = :UserId ORDER BY LastActivityAt DESC", map[string]interface{}{"UserId": userId}); err != nil {
 		return nil, model.NewAppError("SqlSessionStore.GetSessions", "store.sql_session.get_sessions.app_error", nil, err.Error(), http.StatusInternalServerError)
 	}
 
-	rtcs := <-tcs
-	if rtcs.Err != nil {
-		return nil, model.NewAppError("SqlSessionStore.GetSessions", "store.sql_session.get_sessions.app_error", nil, rtcs.Err.Error(), http.StatusInternalServerError)
+	tempMembers, err := me.Team().GetTeamsForUser(userId)
+	if err != nil {
+		return nil, model.NewAppError("SqlSessionStore.GetSessions", "store.sql_session.get_sessions.app_error", nil, err.Error(), http.StatusInternalServerError)
```

Comment:
same here. the err check here should be for GetTeamsForUser instead of getsessions.